zospy/zpcore.py

```diff
@@ -282,13 +282,21 @@ class ZOS:
     ----------
         ZOSAPI (None | netModuleObject): The ZOSAPI interface or if loaded.
         ZOSAPI_NetHelper (None | netModuleObject): The ZOSAPI_NetHelper interface if loaded.
+
+    Parameters
```

Comment:
Please put the Parameters section before the Attributes section, in order to adhere to NumPyDoc: https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes